shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitAnalysisResult.java

```diff
@@ -42,15 +38,6 @@ public ShortCircuitAnalysisResult(List<FaultResult> faultResults, List<LimitViol
         return Collections.unmodifiableList(faultResults);
     }
 
-    /**
-     * The list of limit violations: for instance when the computed short-circuit current on a given equipment is higher
-     * than the maximum admissible value for that equipment. In general, the equipment ID can be completed by the side where
-     * the violation occurs. In a first simple approach, the equipment is a voltage level, and no side is needed.
-     */
-    public List<LimitViolation> getLimitViolations() {
```

Comment:
If we move them to inner classes, it would be nice to keep a convenience method here which concatenates them, so that we can easily get all limit violations at once.

Reply:
Solved, the symmetrical computation API is conserved, but renamed `ShortCircuitAnalysisMultiResult`.